sdk/tests/tutorials/ibor/test_cut_labels.py

```diff
@@ -0,0 +1,231 @@
+import unittest
+from datetime import datetime, date, timedelta
+
+import pytz
+import uuid
+
+import lusid
+import lusid.models as models
+
+from lusid.utilities.api_client_builder import ApiClientBuilder
+from utilities.instrument_loader import InstrumentLoader
+from utilities.test_data_utilities import TestDataUtilities
+from utilities.credentials_source import CredentialsSource
+
+
+class CutLabels(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+
+        cls.scopes_api = lusid.ScopesApi(api_client)
+        cls.portfolios_api = lusid.PortfoliosApi(api_client)
+        cls.transaction_portfolios_api = lusid.TransactionPortfoliosApi(api_client)
+        cls.property_definitions_api = lusid.PropertyDefinitionsApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+        cls.portfolios = lusid.PortfoliosApi(api_client)
+
+        cls.test_data_utilities = TestDataUtilities(cls.transaction_portfolios_api)
+
+        cls.cut_labels = lusid.CutLabelDefinitionsApi(api_client)
+
+    def test_cut_labels(self):
+        
+        def get_guid():
+            return str(uuid.uuid4())[:4]
+
+        # define function to format cut labels
+        def cut_label_formatter(date, cut_label_code):
+            return str(date) + "N" + cut_label_code
+        
+        
+        ## Create cut labels
+        # create a dictionary to keep track of cut labels
+        code = {}
+
+        def create_cut_label(hours, minutes, display_name, description, time_zone):
+            # Create the time for the cut label
+            time = models.CutLocalTime(hours=hours, minutes=minutes)
+            
+            # Define the parameters of the cut label in a request
+            request = models.CutLabelDefinition(
+                code=display_name+"-"+get_guid(), 
+                description=description, 
+                display_name=display_name,
+                cut_local_time=time,
+                time_zone=time_zone)
+
+            # Add the codes of our cut labels to our dictionary
+            code[request.display_name] = request.code 
+            
+            # Send the request to LUSID to create the cut label
+            result = self.cut_labels.create_cut_label_definition(
+                create_request=request)
+
+            # Check that result gives same details as input
+            self.assertEqual(result.display_name, display_name) # should this be self?
```

Comment:
What do you mean here when you say "should this be self?"

Reply:
Whoops, put this in when I was talking to myself and getting used to the syntax, I'll delete it